test/functional/configuration/helpers/viewSelectors.js

```diff
@@ -0,0 +1,129 @@
+    },
+    edgeConfigInputMethodSelectFields: {
+      edgeConfigComboBox: spectrum.comboBox("edgeConfigComboBox"),
+      productionEnvironmentComboBox: spectrum.comboBox(
+        "productionEnvironmentComboBox"
+      ),
+      stagingEnvironmentComboBox: spectrum.comboBox(
+        "stagingEnvironmentComboBox"
+      ),
+      developmentEnvironmentComboBox: spectrum.comboBox(
+        "developmentEnvironmentComboBox"
+      )
+    },
+    edgeConfigInputMethodFreeformFields: {
+      productionEnvironmentTextfield: spectrum.textField(
+        "productionEnvironmentTextfield"
+      ),
+      stagingEnvironmentTextfield: spectrum.textField(
+        "stagingEnvironmentTextfield"
+      ),
+      developmentEnvironmentTextfield: spectrum.textField(
+        "developmentEnvironmentTextfield"
+      )
+    },
+    orgIdField: spectrum.textField("orgIdField"),
+    orgIdRestoreButton: spectrum.button("orgIdRestoreButton"),
+    edgeDomainField: spectrum.textField("edgeDomainField"),
+    edgeDomainRestoreButton: spectrum.button("edgeDomainRestoreButton"),
+    edgeBasePathField: spectrum.textField("edgeBasePathField"),
+    edgeBasePathRestoreButton: spectrum.button("edgeBasePathRestoreButton"),
+    defaultConsent: {
+      inRadio: spectrum.radio("defaultConsentInRadio"),
+      outRadio: spectrum.radio("defaultConsentOutRadio"),
+      pendingRadio: spectrum.radio("defaultConsentPendingRadio"),
+      dataElementRadio: spectrum.radio("defaultConsentDataElementRadio"),
+      dataElementField: spectrum.textField("defaultConsentDataElementField")
+    },
+    idMigrationEnabled: spectrum.checkbox("idMigrationEnabledField"),
+    thirdPartyCookiesEnabled: spectrum.checkbox(
+      "thirdPartyCookiesEnabledField"
+    ),
+    // Due to limitations of the sandbox where tests are run,
+    // testing prehiding style viewing/editing is limited.
+    prehidingStyleEditButton: spectrum.button("prehidingStyleEditButton"),
+    clickCollectionEnabledField: spectrum.checkbox(
+      "clickCollectionEnabledField"
+    ),
+    downloadLinkQualifierField: spectrum.textField(
+      "downloadLinkQualifierField"
+    ),
+    downloadLinkQualifierRestoreButton: spectrum.button(
+      "downloadLinkQualifierRestoreButton"
+    ),
+    downloadLinkQualifierTestButton: spectrum.button(
+      "downloadLinkQualifierTestButton"
+    ),
+    onBeforeEventSendEditButton: spectrum.button("onBeforeEventSendEditButton"),
+    contextGranularity: {
+      allField: spectrum.radio("contextGranularityAllField"),
+      specificField: spectrum.radio("contextGranularitySpecificField")
+    },
+    specificContext: {
+      webField: spectrum.checkbox("contextWebField"),
+      deviceField: spectrum.checkbox("contextDeviceField"),
+      environmentField: spectrum.checkbox("contextEnvironmentField"),
+      placeContextField: spectrum.checkbox("contextPlaceContextField")
+    },
+    deleteButton: spectrum.button("deleteInstanceButton")
+  });
+}
+
+export { instances };
```

Comment:
I split these selectors into a separate file because (1) it was getting rather large and (2) I plan to re-use them in a spec file that tests edge configuration selection.